osu.Game/Rulesets/Judgements/Judgement.cs

```diff
@@ -91,6 +91,12 @@ public virtual HitResult MinResult
             }
         }
 
+        /// <summary>
+        /// Whether the given <see cref="HitResult"/> affects combo in coordination with this <see cref="Judgement"/>.
+        /// </summary>
+        /// <param name="result">The hit result.</param>
+        public virtual bool AffectsCombo(HitResult result) => result.AffectsCombo();
```

Comment:
Uh... when I proposed this flag months ago you brought up that [this has implications on other components that rely on `MaximumStatistics` to compute max combo](https://github.com/ppy/osu/issues/11769#issuecomment-1704773596). So... how are they going to be addressed?

Because as far as I can see

https://github.com/ppy/osu/blob/568bd85533f87321f497ddf07b16050828f63d50/osu.Game/Scoring/ScoreInfoExtensions.cs#L34

is not changed in this pull and becomes straight up incorrect with this change?

Reply:
I... totally forgot about that...